doc/user_guide/user_guide.md

```diff
@@ -49,8 +49,8 @@ As you can see in the example above, you can simply pass the error attributes as
 The predefined table keys are
 
 * `code`: machine-readable unique code (string)
-* `message`: Error description either as static string or as [format with placeholders](https://www.lua.org/manual/5.1/manual.html#pdf-string.format) (string)
-* `parameters`: table of parameter values be used to replace placeholders in the description
+* `message`: Error description either as static string or containing placeholders in double curly brackets.
+* `parameters`: table of parameter values be used to replace placeholders in the message and mitigations
 * `mitigations`: list of hints on how to fix the error (array of strings)
```

Comment:
you could also mention the placeholders here

Reply:
Adde.